Unix/tools/allocdump

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+SCRIPTDIR=$(dirname "$0")
```

Comment:
The `dirname` command isn't necessarily reliable to use for other paths (like on line 7, for `gdb_allocdump`. Although, in this case, it's probably okay.

Why not just hard-code where `installbuilder` will put this, thus playing it "safe"?

Reply:
Switching to hard-coded.